templates/partials/answers/mobilenumber.html

```diff
@@ -0,0 +1,23 @@
+{% from "components/input/_macro.njk" import onsInput %}
+
+{% set input = form.fields[answer.id] if input is undefined else input %}
+
+{% set config = {
+  "id": answer.id,
+  "name": input.name,
+  "value": input._value() | e,
+  "label": {
+    "id": answer.id ~ "-label",
+    "text": answer.label,
+    "description": answer.description
+  },
+  "attributes": {
+    "data-qa": "input-text"
+  },
+  "legend": question_title,
+  "legendClasses": "u-vh",
+  "mutuallyExclusive": mutuallyExclusive,
```

Comment:
We should consider having a test for a mobile number input in a mutually exclusive question.

Reply:
We don't have a use case for this though. Can we not remove this arg from the template until such a time as there is?

Reply:
Yeah, or that.